package.json

```diff
@@ -207,6 +209,7 @@
   "dependencies": {
     "@babel/runtime": "^7.23.1",
     "@babel/runtime-corejs3": "^7.23.1",
+    "@sindresorhus/is": "^4",
```

Comment:
there's already a v6:
https://github.com/sindresorhus/is/releases

but had to use v4 because it's the latest CJS release.

When using ESM modules via `await import` webpack creates a new chunk just for it instead of keeping it in the main bundle:
https://webpack.js.org/guides/code-splitting/#dynamic-imports

I spent a few hours trying to find some workarounds but couldn't get it to inline this into the main bundle so keeping it at v4 until I found a solution for this (will need to for node-fetch v3 (ESM): https://github.com/jsforce/jsforce/pull/1356)